lib/orderbook/OrderBook.ts

```diff
@@ -102,7 +102,8 @@ class OrderBook extends EventEmitter {
           // we must remove the amount that was put on hold while the swap was pending for the remaining order
           this.removeOrderHold(orderId, pairId, quantity);
 
-          await this.persistTrade(swapSuccess.quantity, this.getOwnOrder(swapSuccess.orderId, swapSuccess.pairId), undefined, swapSuccess.rHash);
+          await this.persistTrade(swapSuccess.quantity,
+          this.getOwnOrder(swapSuccess.orderId, swapSuccess.pairId), undefined, swapSuccess.rHash);
```

Comment:
I'd prefer keeping this on one line, but if it's going to be on two lines I think the second line should be indented to be more readable.